package.json

```diff
@@ -17,40 +17,39 @@
     "reselect": "4.0.0"
   },
   "devDependencies": {
-    "@octokit/rest": "15.17.0",
-    "acorn": "6.0.4",
-    "babel-core": "6.26.3",
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
```

Comment:
These semvers don't seem to follow our standard of pinning to a specific single version, so they probably want to be changed.

On a related note, it appears that this patch includes a bunch of changes from the other dev-dependencies patch.  Is there an intended landing order here?

Reply:
Yes this is supposed to land after #4843.